openupgrade_scripts/scripts/event_sms/15.0.1.0/pre-migration.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 ForgeFlow
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+def fill_event_mail_template_ref(env):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE event_mail
+        SET template_ref = 'sms.template,' || sms_template_id
```

Comment:
I think the tests are failing since I merged #3500 this afternoon:

```
  File "/home/runner/work/OpenUpgrade/OpenUpgrade/odoo/addons/event/models/event_event.py", line 439, in _compute_event_mail_ids
    for line in event.event_type_id.event_type_mail_ids
  File "/home/runner/work/OpenUpgrade/OpenUpgrade/odoo/addons/event/models/event_event.py", line 439, in <listcomp>
    for line in event.event_type_id.event_type_mail_ids
  File "/home/runner/work/OpenUpgrade/OpenUpgrade/odoo/addons/event/models/event_mail.py", line 67, in _prepare_event_mail_values
    'template_ref': '%s,%i' % (self.template_ref._name, self.template_ref.id)
AttributeError: 'NoneType' object has no attribute '_name'
```

Would it help if this code is moved to the pre-migration of `event` (after checking that the field exists)?

Reply:
Done!